Makefile

```diff
@@ -31,18 +30,18 @@ default:
 base: base/Dockerfile
 	DOCKER_BUILDKIT=1 docker build $(BUILD_ARCH) -t veracruz/base -f $< .
 
-.PHONY:
+.PHONY: %-build
```

Comment:
Does that really work correctly? The `%` is only on the right-hand side of that rule, which is perhaps interpreted independently of the rule below it. This is definitely not worth worrying about, though. Makefiles are horrible.

Reply:
I am not sure it is working correctly, but adding it had a positive effect.